### PR TITLE
Feature/73940

### PR DESCRIPTION
**Pull request by JKrupinski**
[#73940](https://dev.snowdog.pro/issues/73940)

### CODE REVIEW

File: Magento_Bundle/templates/sales/order/items/renderer.phtml [hunk 1]
```diff
@@ -104,7 +104,7 @@ $prevOptionId = '';
             <?php elseif ($item->getQtyShipped() > 0 && $item->getParentItem() && $block->isShipmentSeparately()) : ?>
                 <?= $block->escapeHtmlAttr(__('Shipped')) ?>:&nbsp;<?= /* @noEscape */ $item->getQtyShipped() * 1; ?>
             <?php else : ?>
-                <?= /* @noEscape */ $parentItem->getQtyOrdered() * 1; ?>
+                <?= $block->escapeHtmlAttr(__('Ordered')) ?>:&nbsp;<?= /* @noEscape */ $parentItem->getQtyOrdered() * 1; ?>
```
Comment (anqaka) [inline Magento_Bundle/templates/sales/order/items/renderer.phtml:107]: shouldn't be here `escapeHtml`? it's not an attribute 🤔

File: Magento_Tax/templates/order/tax.phtml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                    <?php if ($percent !== null) : ?>
+                        (<?= (float) $percent ?>%)
+                    <?php endif; ?>
+                    <br />
+                </td>
+                <td <?= /* @noEscape */ $block->getValueProperties() ?> rowspan="1">
+                    <?= /* @noEscape */ $_order->formatPrice($amount) ?>
+                </td>
+            </tr>
+        <?php endforeach; ?>
```
Comment (anqaka) [inline Magento_Tax/templates/order/tax.phtml:41]: too big indentation

File: Magento_Tax/templates/order/tax.phtml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+<?php endif; ?>
+
+<?php if ($block->displayFullSummary() && $_fullInfo && !$block->getIsPlaneMode()) : ?>
+<tr class="totals-tax-summary">
+<?php elseif ($block->displayFullSummary() && $_fullInfo && $block->getIsPlaneMode()) : ?>
+<tr class="totals-tax-summary plane">
+<?php else : ?>
+<tr class="totals-tax">
+<?php endif; ?>
```
Comment (anqaka) [inline Magento_Tax/templates/order/tax.phtml:50]: missing indentation for `<tr>` tags (lines: 45, 47, 49)

File: Magento_Bundle/templates/sales/order/items/renderer.phtml (hunk 1 above)
Comment (JKrupinski) [reply Magento_Bundle/templates/sales/order/items/renderer.phtml]: fixed

Comment (anqaka) [reply Magento_Bundle/templates/sales/order/items/renderer.phtml]: thanks

**Pull request merged**